config/environments/development.rb

```diff
@@ -4,10 +4,13 @@
   # Settings specified here will take precedence over those in
   # config/application.rb.
 
+  config.en_host = ENV["EN_HOST_NAME"] || "coronavirus-find-support.lndo.site"
```

Comment:
we can remove the lndo specific stuff here and rely on lando setting the ENV vars.

To do this, we create a /lando/.env file and include it from the .lando.yml as per https://docs.lando.dev/config/env.html#default-environment-variables

I can do this if you like?

Reply:
we'll also need to set up these new local addresses in the proxy part of the lando config

Reply:
This is an optional environment variable (in development env only), as this setting will default to  `"coronavirus-find-support.lndo.site"`.   No need to change Lando, I only added it to allow future development testing